frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/CompanyInfo.container.tsx

```diff
@@ -0,0 +1,49 @@
+import useBackendAPI from 'benefit/applicant/hooks/useBackendAPI';
+import React from 'react';
+import { useQuery } from 'react-query';
+
+import CompanyInfo, { CompanyInfoProps } from './CompanyInfo.component';
+
+export type CompanyInfoContainerProps = Omit<CompanyInfoProps, 'data'>;
+
+interface CompanyData {
+  bank_account_number: string;
+  business_id: string;
+  city: string;
+  company_form: string;
+  id: string;
+  name: string;
+  postcode: string;
+  street_address: string;
+}
+
+const CompanyInfoContainer: React.FC<CompanyInfoContainerProps> = (props) => {
+  const { axios, handleResponse } = useBackendAPI();
+  const { isLoading, error, data } = useQuery<CompanyData, { message: string }>(
+    'companyData',
+    async () => {
+      // TODO: replace the hardcoded company ID when auth is implemented
+      const res = axios.get<CompanyData>(`/v1/company/0877830-0`);
+      return handleResponse(res);
+    }
+  );
+
+  const companyData = {
```

Comment:
react-query supports initial data https://react-query.tanstack.com/guides/initial-query-data you could use it instead of this empty value mapping.

Reply:
and maybe it would be nice to move the default/error/loading template objects to some separate mapper file, so all the default "models" can be found easily and be reusable. I bet that we will have components which are reusing the data at some point.